pypeit/core/combine.py

```diff
@@ -114,9 +148,13 @@ def weighted_combine(weights, sci_list, var_list, inmask_stack,
 
     if sigma_clip and nimgs >= 3:
         if sigma_clip_stack is None:
-            msgs.error('You must specify sigma_clip_stack, i.e. which quantity to use for sigma clipping')
+            msgs.error('You must specify sigma_clip_stack; sigma-clipping is based on this array '
+                       'and propagated to the arrays to be stacked.')
         if sigrej is None:
+            # NOTE: If these are changed, make sure to update the doc-string!
             if nimgs <= 2:
```

Comment:
Yeah probably left over from a previous version where nimgs was not being tested at the top.